octoprint_mrbeam/iobeam/iobeam_handler.py

```diff
@@ -263,6 +264,18 @@ def send_analytics_request(self, *args, **kwargs):
         """
         return self._send_command(self.get_request_msg([self.DATASET_ANALYTICS]))
 
+    def send_malfunction_request(self, *args, **kwargs):
```

Comment:
I find the name a bit confusing, like we are requesting a malfunction :sweat_smile: 
What do you think about renaming it to something like `check_for_malfunctions` or `send_malfunctions_present_request`

Reply:
I will rename it to `request_available_malfunctions` what you think about this name?